### DISCUSSION TITLE
After update 2022.605.1 Low frame rate

### DISCUSSION THREAD

**Discussion by weajus** [ppy/osu#18930] [Q&A] [answered]
**Explanation**
After that version, I got text telling me that I'm unable to run full screen, and got lower fps, before that I was getting less delay and my game was running on full screen, is there a way to solve this?, my drivers are in the latest version
Also try the new version with performance boost and still getting lower fps, single thread is more visible around 150fps, and before and on 2022.605.1 it was around 200fps

As bdach said possible https://github.com/ppy/osu/discussions/18814 better report there

**After that version**
You can see that I'm getting lower fps because I'm unable to run full screen, 60-110fps, max 210fps without software or anything that lower the performance

https://user-images.githubusercontent.com/50848502/176257945-720b6758-ec67-48b8-969b-9da09923a8f4.mp4

**Before that version**

Normal Fps 130-170, without software is like 240-480fps or anything that lower the performance

https://user-images.githubusercontent.com/50848502/176258298-a129fa77-005e-4d71-a115-e8dc4918719d.mp4

Specs
DDR3-low 8GB
Inter corei3 6006u 2ghz
Intel hd graphic 520

**Comment by bdach** [comment, marked as answer]
* probably a duplicate of https://github.com/ppy/osu/discussions/18814
* you're not seeing that "exclusive fullscreen" warning before 2022.605.1 because *it didn't exist before*. the warning itself was added in the release you mention ([see changelog](https://osu.ppy.sh/home/changelog/lazer/2022.605.1)). it's not evidence that exclusive fullscreen ever worked before.
